frontend/src/expert/components/SidePanelCard.js

```diff
@@ -184,13 +184,13 @@ function SidePanelCard(props) {
         // Code based on
         //   http://stackoverflow.com/questions/1064089/inserting-a-text-where-cursor-is-using-javascript-jquery
         const code = document.getElementById('code-editor');
-        if (document.selection) {
-            code.focus();
-            var sel = document.selection.createRange();
-            sel.text = text;
+        if (document.selection) { // if text in the code text-field is selected
             code.focus();
+            var sel = document.selection.createRange(); // creates a fragment of the text-field's value
+            sel.text = text; // replaces the fragment with the new text
+            code.focus(); // returns the cursor to the code text-field
```

Comment:
what does this do?

Reply:
why are there two lines of code.focus() ?

Reply:
Not sure. I think it was an unintentional inclusion. We can go ahead and delete it, I think.